grails-app/views/notify/rejectLink.gsp

```diff
@@ -0,0 +1,10 @@
+<p>
+    Your Consent Group <a href="${issueLink}">${issue.summary}</a>,  addition to Project <a href="${values.get("projectLink")}">${values.get("projectSummary")}</a> has been disapproved in the ORSP Portal.
```

Comment:
Super nit (because it is never even seen): double space.
```suggestion
    Your Consent Group <a href="${issueLink}">${issue.summary}</a>, addition to Project <a href="${values.get("projectLink")}">${values.get("projectSummary")}</a> has been disapproved in the ORSP Portal.
```